bin/release

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+output_directory="release_artifacts"
+rm -rf "$output_directory"
+mkdir -p "$output_directory"
+
+function get_version(){
+    awk '/version: ([0-9]\.){2}[0-9]$/' spec/openapi.yml | tr -d "[:alpha:] \n:"
+}
+
+version="$(get_version)"
```

Comment:
if the action runs on tags, we can get the tag using this action:
https://github.com/cyberark/conjur-oss-suite-release/blob/a7fa50b64cc6784a03185f08dbc432f852913d0f/.github/workflows/draft-release.yml#L45-L47

I'd prefer this vs parsing it from the yaml, especially if there's some future reason we want to tag it differently for DAP vs OSS

Reply:
I updated the script to look for an environment variable set by the action.